### ISSUE TITLE
docker sensu stops after few hours

### ISSUE THREAD

**Issue by cwartel** [sensu/sensu-go#3251]
I can launch and use a docker sensu server. Everything's ok with clients and checks but after a few hours (say 10 or so) it stops

## Expected Behavior
It should run till I stop it

## Current Behavior
It stops unexpectedly

## Possible Solution
Not a clue

## Steps to Reproduce (for bugs)
1. I'm on a google hosted Ubuntu 18.04.1 LTS, using a sudoer user sodoing all docker commands 
```
sudo docker run -v /var/lib/sensu:/var/lib/sensu \
-d --name sensu-backend \
-p 3000:3000 -p 8080:8080 -p 8081:8081 sensu/sensu:latest \
sensu-backend start --state-dir /var/lib/sensu/sensu-backend --log-level debug
```

## Context
Here are its last words when `sudo docker logs -f sensu-backend`

```
> panic: runtime error: invalid memory address or nil pointer dereference
> [signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0xe60179]
> 
> goroutine 198 [running]:
> github.com/sensu/sensu-enterprise-go/backend/store/provider.(*Watcher).Disable(0xc003725080)
>         /Users/splourde/go/src/github.com/sensu/sensu-enterprise-go/backend/store/provider/watcher.go:144 +0xb9
> github.com/sensu/sensu-enterprise-go/backend/licensing.applyFeatureList(0xc0001aae10, 0xc002eddcd0, 0x0, 0x0)
>         /Users/splourde/go/src/github.com/sensu/sensu-enterprise-go/backend/licensing/licensing.go:265 +0xdd
> github.com/sensu/sensu-enterprise-go/backend/licensing.(*Enforcer).disableFeatures(0xc00370e800)
>         /Users/splourde/go/src/github.com/sensu/sensu-enterprise-go/backend/licensing/licensing.go:212 +0x91
> github.com/sensu/sensu-enterprise-go/backend/licensing.(*Enforcer).handleWarning(0xc00370e800, 0x0, 0xe7791f700, 0x267da80)
>         /Users/splourde/go/src/github.com/sensu/sensu-enterprise-go/backend/licensing/licensing.go:245 +0x2f6
> github.com/sensu/sensu-enterprise-go/backend/licensing.(*Enforcer).handleResults(0xc00370e800, 0x45cd01)
>         /Users/splourde/go/src/github.com/sensu/sensu-enterprise-go/backend/licensing/licensing.go:140 +0x475
> github.com/sensu/sensu-enterprise-go/backend/licensing.(*Enforcer).Start.func1(0xc00370e800)
>         /Users/splourde/go/src/github.com/sensu/sensu-enterprise-go/backend/licensing/licensing.go:126 +0x2f
> created by github.com/sensu/sensu-enterprise-go/backend/licensing.(*Enforcer).Start
>         /Users/splourde/go/src/github.com/sensu/sensu-enterprise-go/backend/licensing/licensing.go:123 +0x29c
```

## Your Environment

* Sensu version used: I'm not sure, will provide given the command to get tot
* Active Sensu extensions (e.g. WizardVan 0.1.0): none
* Operating System and version (e.g. Ubuntu 14.04): 18.04
* Transport and runtime versions (e.g. RabbitMQ 3.6.2 on Erlang 18.2) : same as first point

Thx for your help

**Comment by ccressent** [comment]
It looks to me like you're coming across a bug that has been fixed in version 5.10.2, which is rather old at this stage. You can check the version of Sensu you're using with the `sensu-backend version` command. With docker that should look something like `docker run -d sensu/sensu sensu-backend version`.

I suggest you update to the latest version. Since you're using the docker image, a simple `docker pull sensu/sensu` should do the trick.

**Comment by cwartel** [comment]
The docker image was indeed older than 5.10.2

```
> sudo docker run --rm sensu/sensu sensu-backend version
sensu-backend version 5.10.1#197b25d, build 197b25d8f21fbee86886304830d87eb79e4a7af5, built 2019-06-26T16:36:38Z
```

Thank you for your help
